### PR TITLE
[SERVICE-394] Standardize Folder Structure + Linting

### PR DESCRIPTION
**Pull request by rypete**
This updates the project to use the "service" folder structuring and runs linting against it.  There are many quick fixes added to let the project compile which should be fixed in normal dev cycle.

### CODE REVIEW

File: scripts/createProviderZip.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+var fs = require('fs');
+var path = require('path');
+var archiver = require('archiver');
+
+var output = fs.createWriteStream(path.resolve(__dirname,'..','dist','provider','layouts-service.zip'));
```
Comment (pjbroadbent) [inline scripts/createProviderZip.js]: `fdc3-service.zip`

File: scripts/server/spawn.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                defaultWidth,
+                defaultHeight
+            },
+            runtime: {
+                arguments: "--v=1" + (realmName ? ` --security-realm=${realmName}${enableMesh ? ' --enable-mesh' : ''}` : ''),
+                version: runtime
+            }
+        };
+        if (useService) {
+            const service = {name: 'layouts'};
```
Comment (pjbroadbent) [inline scripts/server/spawn.js]: `fdc3`

File: README.md [hunk 3]
```diff
@@ -54,49 +63,38 @@ To use the API, you must first include it in your application.
...
 
-* build
- * client.js - the compiled service client
- * demo/ - the demo files
- * provider.js - the compiled service provider
- * ui - the compiled intent/context resolution UI
+The service provider is a standard OpenFin application, only its lifecycle is controlled by the RVM (based upon the requirements of user-launched applications) rather than being launched by users. The provider is deployed to the OpenFin CDN; a zip file is also provided to assist with re-deploying the provider to an alternate location. Direct links to each build are listed in the release notes, available on the [services versions page](https://developer.openfin.co/versions/?product=Services).
 
+## Known Issues
+A list of known issues can be found on our [Versions page.]https://developer.openfin.co/versions/?product=Services
```
Comment (pjbroadbent) [inline README.md]: Fix markup: `[versions page](https://developer.openfin.co/versions/?product=Services).`

File: scripts/createProviderZip.js (hunk 1 above)
Comment (rypete) [reply scripts/createProviderZip.js]: Fixed

File: scripts/server/spawn.js (hunk 2 above)
Comment (rypete) [reply scripts/server/spawn.js]: Fixed

File: README.md (hunk 3 above)
Comment (rypete) [reply README.md]: Fixed

**Review by pjbroadbent — APPROVED** [review]

**Review by ottah — APPROVED** [review]

**Pull request merged**